projects/packages/analyzer/src/Differences/class-class-const-missing.php

```diff
@@ -7,9 +7,10 @@
 
 namespace Automattic\Jetpack\Analyzer\Differences;
 
-use Automattic\Jetpack\Analyzer\PersistentList\Item as PersistentListItem;
 use Automattic\Jetpack\Analyzer\Invocations\Static_Const;
-use Automattic\Jetpack\Analyzer\Warnings\Warning; // TODO - subclasses?
+use Automattic\Jetpack\Analyzer\PersistentList\Item as PersistentListItem;
+use Automattic\Jetpack\Analyzer\Warnings\Warning;
+// TODO - subclasses?
```

Comment:
Feel free to move these TODO comments back up to the Warning lines. They don't make the codesniffer fail, it's just the fixer happens to move them.

Reply:
If the fixer moves them, I'm fine letting them lie.